site/content/posts/2021-06-22-understanding-kubernetes-tests.md

```diff
@@ -0,0 +1,172 @@
+---
+title: Understanding Kubernetes E2E Tests
+image: /img/sonobuoy.svg
+excerpt: Kubernetes testing can be confusing. Let us demystify the test suite for you.
+author_name: John Schnake
+author_url: https://github.com/johnschnake
+categories: [kubernetes, sonobuoy, windows]
+tags: ['Sonobuoy Team', 'John Schnake']
+date: 2021-06-21
+slug: windows-e2e-tests-with-sonobuoy
+---
+
+One of the most common questions that we get from Sonobuoy users is: “Why are so many tests skipped?” Understandably, users want to know that they are running the right set of tests for their Kubernetes clusters but the complexity of the test suite, misnomers, and inaccurate language can make that difficult.
+
+We wanted to discuss the nomenclature of the Kubernetes test suite and discuss when and how to run those tests.
```

Comment:
Do you want to discuss or just share information?

Reply:
I guess its a very one way discussion :) I'll reword.